### ISSUE TITLE
Container size

### ISSUE THREAD

**Issue by ai** [FormidableLabs/react-shuffle#9]
React Shuffle logic is to have hidden block with real controls and visible block for animation with `position: absolute` on top of that hidden block.

But, as result, there is no comtainer size animation, when we delete item.

Example: this is my UX:
![6cijeo0](https://cloud.githubusercontent.com/assets/19343/8775280/4ac2237a-2eef-11e5-8a9f-454a6bc64f8e.png)

When I will delete one of item, “minus” button will move immediately to the left. Because “minus button” is out of Shuffle container and Shuffle container size has no animation.

**Comment by kenwheeler** [comment]
Not sure I understand, video?

**Comment by kenwheeler** [comment]
Oh wait, I know exactly what you are talking about. Does applying a CSS transition declaration fix this?

**Comment by ai** [comment]
Ouh, I fix it in different way. I will test it on new control tomorrow. Is it OK?

**Comment by kenwheeler** [comment]
Sure. It's hard to say exactly how to handle this, because the container could scale vertically or horizontally depending on which tiles were removed. And that's providing that the shuffle container is indeed the layout providing parent.

**Comment by kenwheeler** [comment]
Out of curiosity, is this in use on VK?

**Comment by ai** [comment]
Nope, we use it in Amplifr, “Buffer” for VK and other Russian networks :).

**Comment by kenwheeler** [comment]
Thats awesome. Really cool landing page. :+1:
